cosmos-init/plugins/packages/cosmosc2-demo/plugin.txt

```diff
@@ -17,6 +17,8 @@ VARIABLE example_microservice_name cosmos-example
 VARIABLE templated_microservice_name cosmos-templated
 VARIABLE example_port 9999
 VARIABLE templated_port 5025
+VARIABLE log_retain_time 172800
+VARIABLE reduced_log_retain_time 2592000
```

Comment:
Can you add a comment on what these times are so I don't have to do the math :-)

Reply:
I'll add comments in a future PR.   The numbers are 2 days and 30 days.